docs/data/toolpad/building-ui/canvas-and-inspector.md

```diff
@@ -0,0 +1,43 @@
+# Canvas
+
+<p class="description">
+    Canvas and Inspector are the two main areas when it comes to UI building
+</p>
+
+## Working with canvas
+
+Building UI is as easy as dragging component from the library and dropping it on the canvas.
+
+Once you have a few components on the canvas you can easily tweak them by doing following actions:
+
+<img src="/static/toolpad/canvas-controls.png" width="600px" alt="Canvas controls" />
+
+- Select component - **click** on any component
+- Remove component from canvas - **click** bin icon in the component's hud
```

Comment:
maybe you can order this as:
- selecting
- moving
- deleting

just feels a bit more intuitive to me to have deleting in the end